visual-js/visual/src/utils.ts

```diff
@@ -15,3 +15,8 @@ export const getFullPageConfig: (
   const localCfg = typeof local === 'object' ? local : {};
   return { ...globalCfg, ...localCfg };
 };
+
+export const isSkipMode = (): boolean => {
+  const VISUAL_SKIP_ENV_VAR = 'SAUCE_VISUAL_SKIP';
+  return !!process.env[VISUAL_SKIP_ENV_VAR];
```

Comment:
```suggestion
  return Boolean(process.env[VISUAL_SKIP_ENV_VAR]);
```

more idiomatic (these days)